### PR TITLE
chore(dependencies): Update dependencies

### PR DESCRIPTION
**Pull request by Nhaajt**
+ Update Scala.js to v1.4.0
+ Update stryker4s to v0.10.1
+ Update sbt-ci-release to v1.5.5
+ Update `release.sbt` to use `fullLinkJS` instead of `fullOptJS`
+ Update sbt generate `package.json` task + add keywords

### CODE REVIEW

File: release.sbt [hunk 1]
```diff
@@ -33,11 +33,19 @@ generatePackageJson := s"""{
...
-                          |  "main": "core/target/js-2.13/weapon-regex-opt.js",
+                          |  "main": "core/target/js-2.13/weapon-regex-opt/main.js",
                           |  "repository": {
                           |    "type": "git",
                           |    "url": "${homepage.value.get}"
                           |  },
+                          |  "keywords": [
+                          |    "regex",
+                          |    "regexp",
+                          |    "regular expression",
```
Comment (wijtserekker) [inline release.sbt:44]: Not sure if npm supports two words in a single keyword. Seems like another possibility for the pipeline to fail

**Comment by wijtserekker** [comment]
Other than the comment about the keywords, seems good 👍

File: release.sbt (hunk 1 above)
Comment (Nhaajt) [reply release.sbt:44]: I think npm does support that. Have a look [here](https://nodesource.com/blog/the-basics-of-package-json-in-node-js-and-npm#thekeywordsproperty).

Comment (wijtserekker) [reply release.sbt:44]: Great!

**Review by wijtserekker — APPROVED** [review]
🔥

**Pull request merged**